packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeProvider.cs

```diff
@@ -333,6 +424,429 @@ private MethodProvider BuildEmptyConstructor()
                 this);
         }
 
+        /// <summary>
+        /// Produces the validation body statements for the JSON serialization format.
+        /// </summary>
+        private MethodBodyStatement CreateValidateJsonFormat(CSharpType modelInterface, SerializationFormatValidationType validationType)
```

Comment:
Lets replace `SerializationFormatValidationType validationType` with `string action`, the extra layer of conversion isn't needed since we are never switching over this enum.